packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2461,6 +2463,7 @@ const findBookingQuery = async (bookingId: number) => {
           name: true,
           email: true,
           timeZone: true,
+          username: true,
```

Comment:
Had to add the username to the query since it was previously only getting name, email, and timezone for the organizer